### DISCUSSION TITLE
NFL Network

### DISCUSSION THREAD

**Discussion by Aurora1975** [iptv-org/community#864] [Q&A]
### Channel ID

NFLNetwork.us

### Stream URL

https://nflwebcast.com/live/nflnetwork.html

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by CoolCanuck** [comment]
I don't think that's an official site

NFL Network is available on Xumo in USA but looks heavily tokenized?

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/us_moveonjoy.m3u#L54-L55
